code/game/dna/genes/powers.dm

```diff
@@ -177,6 +175,15 @@
 		else
 			to_chat(T,"<span class='notice'>You hear a voice inside your head: [message] </span>")
 		to_chat(user,"<span class='notice'>Projected to <b>[T]</b>: [message]</span>")
+		if(all_switch && targets.len > 1)
+			all_switch = FALSE
+			for(var/mob/dead/observer/G in dead_mob_list)
+				G.show_message("<i>Telepathy, <b>[user]</b> to everyone nearby</b>: [message]</i>")
```

Comment:
you could just do this with `[english_list(targets)]`, which would parse as:

`Ian Sanity to Kiyiyichayayichaka, Helvella Geophylla, Faguette, Chris Haverson, Carver Buttocks, Yayiyiyi, and Ian Sanity: come look`

This is arguably better for logging purposes, but on the other hand, we don't normally log who can hear nearby chatter anyway, so your solution also seems perfectly reasonable. I'm presenting this as a non-mandatory alternative.